lib/hyperwallet/resources/payee.rb

```diff
@@ -29,8 +29,9 @@ def show(token: nil)
         handle_response
       end
 
-      def update(token:, attributes_to_update: )
-        connector.put(resource: ENDPOINT + "/" + token, payload: JSON.generate(prepare_payload(payload_attributes: attributes_to_update)))
+      def update(attributes:, token: nil)
```

Comment:
Though I like the name change, changing it from `attributes_to_update` to `attributes` means that an app using this gem would suddenly break after calling `update`. If we are sure that this method is not being used then it might be ok.

Reply:
That's a good point; I don't think the method is being used, though it may cause issues in the future if we need to use it. Possibly the best is to keep it as flexible as possible.